### ISSUE TITLE
Docs: Can we update database-tables.png to include the views?

### ISSUE THREAD

**Issue by odscjames** [open-contracting/kingfisher-process#182]
From @jpmckinney in #150

**Comment by duncandewhurst** [comment]
Just to clarify, do you mean the views in the main kingfisher process schema (e.g. `release_with_collection` etc.) or everything the `views` schema (e.g. `views.release_summary` etc.)?

I think the former should be added to the diagram in the kingfisher-process documentation but the latter should live in kingfisher-views documentation.

@mrshll1001 @pindec may have views on whether a diagram in the kingfisher-views docs would be useful.

**Comment by duncandewhurst** [comment]
![](https://github.trello.services/images/mini-trello-icon.png) [Improve docs of kingfisher-views and kingfisher-process database](https://trello.com/c/3N0ylJE5/96-improve-docs-of-kingfisher-views-and-kingfisher-process-database)

**Comment by jpmckinney** [comment]
I just meant the former.

**Comment by pindec** [comment]
Agree that the former would be helpful.

Re: kingfisher-views: I don't think we _need_ a diagram for the kingfisher-views database structure since the tables are mostly not directly related to each other.  

However, a diagram showing what summary tables are available for which stages of the contracting process, and how the indices are constructed, would be helpful for anyone starting to use kingfisher-views. For example, how values in `contract_documents_summary.contract_index` and `contract_index.document_index` relate to items in the respective arrays.

**Comment by odscjames** [comment]
I thought this would be an easy and quick regenerate, so I looked at this but it turns out it's not. SchemaSpy is just not including those views in the diagram. It maybe is because there are no foreign keys defined, but they don't appear on the Orphans page either. I took the opportunity to write up instructions tho. 

Not sure how long this will take to sort out, so a discussion about priority may be wise.

**Comment by jpmckinney** [comment]
Maybe just add before the diagram:

```rst
The diagram below excludes the :ref:`release, record and compiled_release views with added collection information<with-collection-views>`.
```

And then before that header:

```rst
.. _with-collection-views:
```

**Comment by jpmckinney** [comment]
Closed by #224.
